### DISCUSSION TITLE
[BP#1] [Proposition] Reformuler le 1er paragraphe, car les sources citées sont trop faibles

### DISCUSSION THREAD

**Discussion by florimondmanca** [cnumr/best-practices#453] [Ideas]
## Proposition

Dans la BP001, je propose de remplacer ce paragraphe  (en gras la partie qui change) :

> **Plusieurs études (Cast Software et Standish Group, notamment) démontrent que 70 % des fonctionnalités demandées par les utilisateurs ne sont pas essentielles et que 45 % ne sont jamais utilisées.** En réduisant la couverture et la profondeur fonctionnelle de l’application, on abaisse son coût de développement initial, sa dette technique et les impacts environnementaux associés.

Par (en gras la nouvelle rédaction) :

> **Dans une application, une partie des fonctionnalités peut s'avérer peu voire jamais utilisées.** En réduisant la couverture et la profondeur fonctionnelle de l’application, on abaisse son coût de développement initial, sa dette technique et les impacts environnementaux associés.

## Détails

Bonjour,

Je me permets de vous soumettre un billet de blog que j'ai écrit au sujet d'un chiffre que l'on avait pour habitude d'utiliser dans nos présentations, et qui me semble trompeur : 

["45% des fonctionnalités jamais utilisées" : qu'en est-il vraiment ?](https://fairness.coop/blog/2023/45-pourcent-fonctionnalites-jamais-utilisees/)

Le résultat de mon enquête et la conclusion de l'article m'amènent à proposer le changement ci-dessus.

En résumé, il semble que les sources que l'on peut retrouver (je n'ai trouvé aucune source véritablement primaire) soient trop faibles pour dire que "plusieurs études démontrent que". Selon moi, il faut affaiblir la formulation.

En effet, d'une part je n'ai pu retrouver aucune publication primaire citée, ni de Cast Software (je n'ai rien trouvé), ni de Standish Group (seulement une slide lors d'un talk donné par le président de Standish Group à l'XP Conference de 2002, comportant un graphique sans aucune référence précise).

J'ai même trouvé des sources contradictoires sur les données sur laquelle s'appuierait ce genre de chiffre :

* Quelqu'un du Standish Group aurait informé un blogueur ayant mené sa propre enquête ([Are 64% of Features Really Rarely or Never Used? ](https://www.mountaingoatsoftware.com/blog/are-64-of-features-really-rarely-or-never-used), Mike Cohn, 2015) que l'étude en question aurait été faite "sur 4 applications internes".
* Une autre source Standish Group que j'ai retrouvée ([Modernization](https://www.standishgroup.com/sample_research_files/Modernization.pdf, rapport client Standish Group, 2010, 1.6 Mo) citait "une étude de 1996" qui comparait, pour "_100 custom development applications_", un inventaire des fonctionnalités avec ce que les utilisateurs utilisaient vraiment.

Ma proposition est donc de retirer la référence à ces études. On se retrouve sans chiffre pour appuyer le propos ; on a à la place un truisme logique qui semble bien plus robuste : "_une partie_ des fonctionnalités peut s'avérer peu ou jamais utilisée" (sous-entendu : la taille de cette part peut varier). Cela me semble préférable à un chiffre qu'il faudrait peut-être qualifier de faux en l'état.

Je suis à disposition pour en discuter davantage. :smiley:

**Comment by ldevernay** [comment]
Je fais remonter cette discussion car il me semble important :
* D'envisager la reformulation de la première partie de la BP
* De mentionner les sources utilisées
